roles/openshift_aws/defaults/main.yml

```diff
@@ -46,6 +47,45 @@ openshift_aws_ami_tags:
 openshift_aws_s3_mode: create
 openshift_aws_s3_bucket_name: "{{ openshift_aws_clusterid }}-docker-registry"
 
+openshift_aws_dns_dict:
+  external:
+    provider: ''  # [ dyn, route53 ]
```

Comment:
create a top-level variable to reference here so user can just set that instead of having to stick this whole dictionary in their inventory to define one thing.